CHANGELOG.md

```diff
@@ -1,3 +1,184 @@
+## v5.7.0 (2018-02-20):
+
+Hey y'all, it's been a while.  Expect our release rate to increase back to
+normal here, as we've got a lot in the pipeline.  Right now we've got a
+bunch of things from folks at npm.  In the next release we'll be focusing on
+user contributions and there are a lot of them queued up!
+
+This release brings a bunch of exciting new features and bug fixes.
+
+### PACKAGE-LOCK GIT MERGE CONFLICT RESOLUTION
+
+Allow `npm install` to fix `package-lock.json` and `npm-shrinkwrap.json`
+files that have merge conflicts in them without your having to edit them.
+It works in conjunction with
+[`npm-merge-driver`](https://www.npmjs.com/package/npm-merge-driver) to
+entirely eliminate package-lock merge conflicts.
+
+* [`e27674c22`](https://github.com/npm/npm/commit/e27674c221dc17473f23bffa50123e49a021ae34)
+  Automatically resolve merge conflicts in lock-files.
+  ([@zkat](https://github.com/zkat))
+
+### NPM CI
+
+The new `npm ci` command installs from your lock-file ONLY.  If your
+`package.json` and your lock-file are out of sync then it will report an error.
+
+It works by throwing away your `node_modules` and recreating it from scratch.
+
+Beyond guaranteeing you that you'll only get what is in your lock-file it's
+also much faster (2x-10x!) than `npm install` when you don't start with a
+`node_modules`.
+
+As you may take from the name, we expect it to be a big boon to continuous
+integration environments.  We also expect that folks who do production
+deploys from git tags will see major gains.
+
+* [`5e4de9c99`](https://github.com/npm/npm/commit/5e4de9c99c934e25ef7b9c788244cc3c993da559)
+  Add new `npm ci` installer.
+  ([@zkat](https://github.com/zkat))
+
+### OTHER NEW FEATURES
+
+* [`4d418c21b`](https://github.com/npm/npm/commit/4d418c21b051f23a3b6fb085449fdf4bf4f826f5)
+  [#19817](https://github.com/npm/npm/pull/19817)
+  Include contributor count in installation summary.
+  ([@kemitchell](https://github.com/kemitchell))
+* [`17079c2a8`](https://github.com/npm/npm/commit/17079c2a880d3a6f8a67c8f17eedc7eb51b8f0f8)
+  Require password to change email through `npm profile`.
+  ([@iarna](https://github.com/iarna))
+* [`1d385ac0c`](https://github.com/npm/npm/commit/1d385ac0c35380b43fb5e453805f4850326902c0)
```

Comment:
I would omit this one -- there was more stuff that was needed to get this working right, and `pacote@7.4.0` (the kinda-broken one) is the one that actually finishes adding all the stuff that needs to be there. Not to mention, I kinda removed most of the code from this commit as part of `npm-ci` integration. 🙈